bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/internal/MDNSClientImpl.java

```diff
@@ -226,4 +227,8 @@ public void onChanged(List<CidrAddress> added, List<CidrAddress> removed) {
             createJmDNSByAddress(address);
         }
     }
+
+    @Override
+    public void onPrimaryAddressChanged(@Nullable String oldPrimaryAddress, @Nullable String newPrimaryAddress) {
+    }
```

Comment:
Maybe add a comment here that this is intentionally left empty because this implementation is not interested in this information.